e2e/tests/nightly/admin/user-profile-admin.spec.ts

```diff
@@ -0,0 +1,469 @@
+
+    const saveButton = userProfileAdminPage.getSaveButton();
+    await verifyButtonEnabled(saveButton, true);
+    await saveButton.click();
+    await userProfileAdminPage.waitForLoad();
+
+    const newStatus = await statusCell.getCellValue();
+    if (oldStatus === 'Bypassed') {
+      expect(newStatus).toBe('Incomplete');
+      expect(await userProfileAdminPage.getDataAccessTiers()).toEqual('Registered Tier');
+    } else {
+      expect(newStatus).toBe('Bypassed');
+      expect(await userProfileAdminPage.getDataAccessTiers()).toEqual('Registered Tier');
+    }
+
+    if (newStatus === 'Incomplete') {
+      await bypassSwitch.turnOn();
+      await saveButton.click();
+      await userProfileAdminPage.waitForLoad();
+    }
+  });
+
+  test('Can modify editable information', async () => {
+    const userProfileAdminPage = new UserProfileAdminPage(adminTab);
+    await userProfileAdminPage.waitForLoad();
+
+    // Modify Institution Role
+    const institutionRole = userProfileAdminPage.getInstitutionalRole();
+    const oldRole = await institutionRole.getSelectedValue();
+
+    let options = (await institutionRole.getAllOptionTexts()).filter((role) => role !== oldRole);
+    const randRole = fp.shuffle(options)[0];
+    await institutionRole.select(randRole);
+
+    // Modify Initial Credit Limit
+    const initialCreditLimit = userProfileAdminPage.getInitialCreditLimit();
+    const oldCreditLimit = await initialCreditLimit.getSelectedValue();
+
+    options = (await initialCreditLimit.getAllOptionTexts()).filter((role) => role !== oldCreditLimit);
+    const randCreditLimit = fp.shuffle(options)[0];
+    await initialCreditLimit.select(randCreditLimit);
+
+    // Initial Credit Limit remains unchanged before save
+    const creditLimitNumber = parseInt(parseForNumbericalString(await initialCreditLimit.getSelectedValue())[0]);
+    expect(isNaN(creditLimitNumber)).toBeFalsy();
+    expect(creditLimitNumber).not.toEqual(parseInt(parseForNumbericalString(oldCreditLimit)[0]));
+
+    // Save and verify changes
+    const saveButton = userProfileAdminPage.getSaveButton();
+    await saveButton.click();
+    await waitWhileLoading(adminTab);
+    await saveButton.waitUntilDisabled();
+
+    const newRole = await institutionRole.getSelectedValue();
+    expect(newRole).toEqual(randRole);
+
+    const newCreditLimit = await initialCreditLimit.getSelectedValue();
+    expect(newCreditLimit).toEqual(randCreditLimit);
+  });
+});
+
+async function verifyButtonEnabled(button: Button, enabled: boolean): Promise<void> {
+  expect(await button.isCursorNotAllowed()).toBe(!enabled);
+}
+
+async function verifyToggleSwitchNewBackgroundColor(page: Page, cell: Cell): Promise<void> {
+  const divXpath = `${cell.getXpath()}/div`;
+
+  const background = await getStyleValue<string>(page, await page.waitForXPath(divXpath), 'background-color');
+
+  expect(background).toEqual('rgb(248, 201, 84)');
```

Comment:
this would make a good const, like CHANGED_COLOR or something

Reply:
yes, `const CHANGED_BACKGROUND_COLOR`